pkg/landscape/lib/gladio.hoon

```diff
@@ -78,6 +76,18 @@
   .^(* (scry %graph-store /export/noun))
 ++  associations
   ~+  .^([@ =associations:met ~] (scry %metadata-store /export/noun))
+++  my-channels-associations
+  =/  assoc
+    .^(associations:met %gx [(scot %p our.bowl) %metadata-store (scot %da now.bowl) %associations %noun ~])
+  %-  ~(gas by *associations:met)
+  %+  skim
+      ~(tap by assoc)
+    |=  [m=md-resource:met [g=resource:res metdat=metadatum:met]]
+    ?&  =(| hidden.metdat)
+        =(name.g name.resource.m)
+        =(entity.g our.bowl)
+        =(%graph -.config.metdat)
```

Comment:
I think you want this to also make sure that it's not a chat so you don't pick up the dms. not sure if this will compile since it's a tagged union type:

```suggestion
        &(=(%graph -.config.metdat) !=(%chat module.config.metdat))
```